src/css/style.css

```diff
@@ -688,3 +688,118 @@ textarea {
     background-position: top right;
   }
 }
+
+/* Modal styles */
+.modal-container {
+  display: none;
+}
+
+.modal {
+  position: fixed;
+  background-color: rgba(128, 128, 128, 0.9);
+  inset: 0;
+  z-index: 10000;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.modal-box {
+  max-width: min(700px, 90%);
+  background-color: var(--color-white);
+  border-radius: 8px;
+  padding: var(--size-300);
+  overflow-y: auto;
+}
+
+.modal-header {
```

Comment:
- [ ] For the mobile view of the modal pop-up, kindly foloow the color and the alignment of the heading and the close button.

Your design
![image](https://user-images.githubusercontent.com/81932478/188737221-9397b08a-9696-4acb-ad2f-e5a1c88c0e5b.png)

Figma design
![image](https://user-images.githubusercontent.com/81932478/188737301-91e09b52-0f55-4f66-95f9-4b2b72546b4e.png)